### PR TITLE
Update filtering rule

### PR DESCRIPTION
**Pull request by MarionJHolloway**
# Description
Update filtering rule to account for data that is windsorised, rather than nulled.

# How to test
Unit test passing
Run in branch successfully: https://eu-west-2.console.aws.amazon.com/states/home?region=eu-west-2#/v2/executions/details/arn:aws:states:eu-west-2:344210435447:execution:update-filtering-rule-Ind-CQC-Filled-Post-Estimates-Pipeline:092bd2eb-c7e5-4c6b-9b13-9cc0a2c31216

# Developer checklist
- [x] Unit test
- [x] Linked to Trello ticket: https://trello.com/c/kp8RLn7V/768-identify-values-changed-by-filtering-winsorizing
- [x] Documentation up to date

### CODE REVIEW

File: tests/test_file_schemas.py [hunk 1]
```diff
@@ -1909,6 +1909,7 @@ class WinsorizeCareHomeFilledPostsPerBedRatioOutliersSchema:
             StructField(IndCQC.primary_service_type, StringType(), True),
             StructField(IndCQC.number_of_beds, IntegerType(), True),
             StructField(IndCQC.ascwds_filled_posts, DoubleType(), True),
+            StructField(IndCQC.ascwds_filled_posts_dedup, DoubleType(), True),
```
Comment (garypolzin) [inline tests/test_file_schemas.py:1912]: There's no point changing it all back but you could have saved a bit of time/effort and just renamed `ascwds_filled_posts` to `ascwds_filled_posts_dedup` and left all the data as it was as we don't actually use `ascwds_filled_posts` - also slightly ironic really that you duplicated everything to create the dedup column!

File: utils/column_values/categorical_column_values.py [hunk 2]
```diff
@@ -528,3 +528,4 @@ class AscwdsFilteringRule(ColumnValues):
     care_home_location_was_grouped_provider: str = (
         "care_home_location_was_grouped_provider"
     )
+    winsorised_beds_ratio_outlier: str = "winsorised_beds_ratio_outlier"
```
Comment (garypolzin) [inline utils/column_values/categorical_column_values.py:531]: I tried really hard but could find any reference online to `winsorised` (with an 's') so I went with `winsorized` (with an 'z') to match everyone else (much prefer an 's' though)

also takes all my strength to not put a 'd' in 'windsorize'

**Review by garypolzin — APPROVED** [review]
looks good, I'll approve now as the only change is spelling :)

File: tests/test_file_schemas.py (hunk 1 above)
Comment (MarionJHolloway) [reply tests/test_file_schemas.py:1912]: Oh yeah... oops.

**Pull request merged**